csireverseproxy/main.go

```diff
@@ -58,6 +58,9 @@ type ServerOpts struct {
 	ConfigDir      string
 	ConfigFileName string
 	InCluster      bool
+	SecretFilePath string
+	SecretName     string
```

Comment:
The reverse proxy doesn't need to know the name of the Secret. It will only need to know the location of the mount path and use that as the source of truth for the credentials, endpoints, etc. Viper should monitor this mount path and doesn't need to monitor the Secret.

Reply:
My bad. the SecretName was used earlier but it is not referenced anymore. I will remove SecretName. SecretFilePath also not being used, will remove. The RP monitors the configFile  which is the path+filename  of the mounted secret as you pointed out.

Reply:
fixed